package.json

```diff
@@ -72,7 +72,7 @@
     "prepare-for-release": "terra prepare-for-release",
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "start": "tt-serve",
+    "start": "webpack-dev-server",
     "start-heroku": "tt-serve-static --port $PORT --site './build'",
     "start-prod": "tt-serve --env.disableHotReloading -p",
     "start-static": "tt-serve-static --site ./build",
```

Comment:
we can use terra cli to execute wdio now
```json
"wdio-default": "terra wdio",
"wdio-lowlight": "terra wdio --themes clinical-lowlight-theme"

Reply:
updated here https://github.com/cerner/terra-core/pull/3348/commits/55f850fda50e1ad65c4415da4c5fdeec2d1cd538#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R80